internal/testhelper/testhelper.go

```diff
@@ -67,8 +67,6 @@ func CreateFullModSLAMServiceIntegration(
 	timedSensor s.TimedSensor,
 	logger golog.Logger,
 ) (slam.Service, error) {
-	t.Helper()
```

Comment:
Why remove these?

Reply:
Because I misunderstood what `t.Helper()` did. I thought they provided more when a test fails within a function annotated with `t.Helper()`. In point of fact it actually does the opposite. If any `test.That` fails within a function in which `t.Helper()` was called, the log that is emitted, is not on the `test.That` line, but rather the line in which the function the `t.Helper` was called within.